filesystem-repository/src/main/java/com/tutrit/filesystem/persistence/FilesystemCarPersistence.java

```
@@ -0,0 +1,23 @@
+package com.tutrit.filesystem.persistence;
+
+import com.tutrit.persistence.core.bean.Car;
+import com.tutrit.persistence.core.persistence.CarPersistence;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class FilesystemCarPersistence implements CarPersistence {
+    private static final List<Car> cars = new ArrayList<>();
+
+    @Override
+    public Car save(Car car) throws RuntimeException {
+        return null;
```

make custom `NotImplementedException` and throw it instead of returning null.

---

I created customer exception and throw it